packages/pwa/README.md

```diff
@@ -29,9 +27,23 @@ Now that the development server is running, you can open a browser and preview y
 
 See the [Localization README.md](./app/translations/README.md) for important setup instructions for localization.
 
-## URL Customization
+## Retail Project Configurations
+
+By default the Retail React App's configuration is located within the `app/config` folder, and more specifically the `default.js` file. Within this file you can make changes to how your application runs on the Managed Runtime via your SSR options like `ssrParameters`, `pageNotFoundURL`, etc. Similarly you can make changes to how your Retail React App works by changing the configuration options under the `app` key. Some of those options are your Commerce API and Einstein API connection strings, your URL settings, and site and site alias configurations.
+
+We have taken it upon ourselves to extract the sites configured for you application into its own file called `config/sites.js`. This will allow you to more easily synchronize the sites supported by your Retail React App and those defined in your Business Manager backend. But you can define your sites directly in the main configuration file is you so choose. 
```

Comment:
```suggestion
We have taken it upon ourselves to extract the sites configured for you application into its own file called `config/sites.js`. This will allow you to more easily synchronize the sites supported by your Retail React App and those defined in your Business Manager backend, but you can opt to define your sites directly in the main configuration file
```